### PR TITLE
Feat: Addded result page

### PR DESCRIPTION
**Pull request by neetukumari4858**
Feat: Addded result page
Screenshot from 2022-02-17 10-31-39
Screenshot from 2022-02-17 10-32-14
Screenshot from 2022-02-17 10-33-52

### CODE REVIEW

**Review by Kothalkarpriya — APPROVED** [review]
Great work! You can merge

**Comment by Kothalkarpriya** [comment]
Just host it somewhere , we would be able to see the deploy version too

File: Result/result1.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    color: var(--black-color);
+    font-weight: bold;
+    background-color: var(--gainsboro-color);
+}
+.Answer-currect{
```
Comment (ShreyashMahajan) [inline Result/result1.css]: spelling of correct is not correct here

**Review by ShreyashMahajan — APPROVED** [review]
UI is looking awesome. It would be good if a preview would have been available. Good to merge

File: Result/result1.css (hunk 1 above)
Comment (neetukumari4858) [reply Result/result1.css]: ohh by mistake it happend i did't saw this thx for remembering to me .

Comment (neetukumari4858) [reply Result/result1.css:51]: ohh by mistake it happend i did't saw this thx for remembering to me .

Comment (neetukumari4858) [reply Result/result1.css:51]: ohh by mistake it happend i did't saw this thx for remembering to me .

Comment (neetukumari4858) [reply Result/result1.css:51]: ohh by mistake it happend i did't saw this thx for remembering to me .

**Pull request merged**